deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Common Labels
+labels:
+  app: "adapter-master-data"
+  group: "rainmaker"
+
+# Ingress Configs
+ingress:
+  enabled: true
+  zuul: false
```

Comment:
When zuul is marked false, the api call from outside will directly reach the server. So standard protections like, Authn&AuthZ, Rate Limiting, etc. will get skipped, due to which there are chances of an attack.